__tests__/app.test.js

```diff
@@ -107,6 +107,71 @@ describe('/api/articles/:article_id', () => {
             })
         })
     })
+    describe('PATCH', () => {
+        test('PATCH 200: increments the votes of the speficied article if the value in inc_votes is positive and responds with the udpated article', () => {
+            return request(app)
+            .patch("/api/articles/1")
+            .send({inc_votes: 3})
+            .expect(200)
+            .then(({body}) => {
+                expect(body.article).toMatchObject({
+                    article_id: 1,
+                    title: "Living in the shadow of a great man",
+                    topic: "mitch",
+                    author: "butter_bridge",
+                    body: "I find this existence challenging",
+                    created_at: expect.any(String),
+                    votes: 103,
+                    article_img_url: "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+                })
+            })
+        })
+        test('PATCH 200: decrements the votes of the speficied article if the value in inc_votes is negative and responds with the udpated article', () => {
+            return request(app)
+            .patch("/api/articles/1")
+            .send({inc_votes: -20})
+            .expect(200)
+            .then(({body}) => {
+                expect(body.article).toMatchObject({
+                    article_id: 1,
+                    title: "Living in the shadow of a great man",
+                    topic: "mitch",
+                    author: "butter_bridge",
+                    body: "I find this existence challenging",
+                    created_at: expect.any(String),
+                    votes: 80,
+                    article_img_url: "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+                })
+            })
+        })
+        test('PATCH 400: responds with an error status and a relevant message when attempting to update an article with a request body that does not contain the correct fields', () => {
+            return request(app)
+            .patch('/api/articles/1')
+            .send({})
+            .expect(400)
+            .then(({body}) => {
+                expect(body.msg).toBe("Bad request")
+            })
+        })
```

Comment:
This is getting a bit more into the finer details of best practice when writing tests...

Would we want this request to give back an error ? 🤔 

If we assume that we could want to expand this endpoint one day, to be able to PATCH the `body` property say. If we don't provide an `inc_votes` property it would give back an error, even though we might be trying to PATCH something different.
Might it be better for future expansion to have this request that is missing `inc_votes` to respond with `200` and an unchanged article ?